module/Checker/config/module.config.php

```diff
@@ -16,8 +17,14 @@
             'check:discharges' => CheckDischargesCommand::class,
             'check:members:keys' => CheckAuthenticationKeysCommand::class,
             'check:membership:expiration' => CheckMembershipExpirationCommand::class,
+            'check:membership:renewal:graduate' => CheckMembershipGraduateRenewalCommand::class,
```

Comment:
.

Reply:
I see I actually didn't type anything here, but I was thinking `check:membership:graduate:renewal` but let's not keep changing things.